src/analyze/gui/middleelide.cpp

```diff
@@ -0,0 +1,42 @@
+#include "middleelide.h"
+#include <QChar>
+
+MiddleElide::MiddleElide()
+{
+
+}
+
+QString MiddleElide::elideAngleBracket(const QString& s)
+{
+   return substituteAngleBrackets(s);
+}
+
+QString MiddleElide::substituteAngleBrackets(const QString& s)
```

Comment:
this should just be a free function within `analyze/gui/util.h` - there's no need to put this into a separate file and into a class namespace

Reply:
I was not sure if util.h is the right place for the method.
But it is ok for me